x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -139,48 +139,52 @@ export const TagTable: FC<TagTableProps> = ({
       : []),
   ];
 
+  const testSubjectState = !loading ? 'table-is-ready' : 'table-is-loading';
+
   return (
-    <EuiInMemoryTable
-      data-test-subj="tagsManagementTable"
-      ref={tableRef}
-      childrenBetween={actionBar}
-      loading={loading}
-      itemId={'id'}
-      columns={columns}
-      items={tags}
-      pagination={tablePagination}
-      sorting={sorting}
-      tableCaption={i18n.translate('xpack.savedObjectsTagging.management.table.columns.caption', {
-        defaultMessage: 'Tags',
-      })}
-      rowHeader="name"
-      selection={
-        allowSelection
-          ? {
-              initialSelected: selectedTags,
-              onSelectionChange,
-            }
-          : undefined
-      }
-      search={{
-        defaultQuery: initialQuery,
-        onChange: ({ query }) => {
-          onQueryChange(query || undefined);
-        },
-        box: {
-          'data-test-subj': 'tagsManagementSearchBar',
-          incremental: true,
-          schema: {
-            fields: {
-              name: { type: 'string' },
-              description: { type: 'string' },
+    <div data-test-subj={testSubjectState}>
```

Comment:
nit: Can we avoid additional div here? I assume the this subj can be added to the <EuiInMemoryTable data-test-subj="tagsManagementTable ${testSubjectState}"

Reply:
Pushed 5485830a9474f21d508ab747630cdaa9f8cb4715